cmd/stupid/main.go

```diff
@@ -20,60 +21,97 @@ func main() {
 			logger.SetLevel(lvl)
 		}
 	}
-	if len(os.Args) != 3 {
-		fmt.Printf("Usage: stupid config.yaml 500\n")
+	err := process(logger)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
where and how exactly do you plan to log errors??

Reply:
en, make logger.Error in main.go

Reply:
couldn't session capture `logger.Errorf`?